src/models/Request.js

```diff
@@ -0,0 +1,15 @@
+export default (sequelize, DataTypes) => {
+  const Request = sequelize.define('requests', {
+    status: {
+      type: DataTypes.ENUM('pending', 'rejected', 'approved'),
+      allowNull: false,
+      defaultValue: 'pending',
+    },
+  }, {});
+  Request.associate = (models) => {
+    Request.hasMany(models.comments, { as: 'requestComments', foreignKey: 'reqId' });
```

Comment:
Please let the change reflect in all references to the values so there would be no error